docs/Silver Blaze.html

```diff
@@ -14,1391 +14,2546 @@ <h1>
 	</h1>
 	<p>
 		I am afraid, Watson, that I shall have to go,” said Holmes, as we
+I am afraid, Watson, that I shall have to go,” said Holmes, as we
 		sat down together to our breakfast one morning.
+sat down together to our breakfast one morning.
 	</p>
 	<p>		
```

Comment:
Lines throughout the 'docs/Silver Blaze.html' file seems to have doubled. Did you mean to commit these changes?